lib/task-graph.js

```diff
@@ -360,7 +365,15 @@ function taskGraphFactory(
             assert.arrayOfObject(self.definition.tasks, 'Graph.tasks');
             return Promise.map(self.definition.tasks, function(taskData) {
                 if (!_.has(taskData, 'taskDefinition')) {
-                    return store.getTaskDefinition(taskData.taskName)
+                    var taskName = taskData.taskName;
```

Comment:
duplicated code with above, please extract a function if possible

Reply:
done,  however this line is in the new function.